docs/architecture.md

```
@@ -0,0 +1,159 @@
+│       ├── presentation/  # pages/, cubits/, widgets/, validators/
+│       └── support/       # feature-internal helpers
+├── uikit/
+│   ├── buttons/ dialogs/ images/ inputs/ cards/ menus/ themes/
+└── main.dart → runner.dart  (single entrypoint, no flavors)
+```
+
+## Layers
+
+Each feature has three layers; each layer only knows about what is below it:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)